dynamic_instrumentation/src/point.rs

```diff
@@ -0,0 +1,199 @@
+use c2rust_analysis_rt::mir_loc::{self, EventMetadata, TransferKind};
+use rustc_index::vec::Idx;
+use rustc_middle::{
+    mir::{Body, Location, Place, Rvalue},
+    ty::TyCtxt,
+};
+use rustc_span::{def_id::DefId, Symbol};
+
+use crate::{
+    arg::{ArgKind, InstrumentationArg},
+    instrument::find_instrumentation_def,
+    into_operand::IntoOperand,
```

Comment:
`into_operand` can be folded into here, it's not used elsewhere and is specific to instrumentation points